doc/cli/operator-sdk_cleanup.md

```diff
@@ -21,6 +21,7 @@ operator-sdk cleanup [flags]
       --install-mode string       [olm only] InstallMode to create OperatorGroup with. Format: InstallModeType=[ns1,ns2[, ...]]
       --include strings           [olm only] Path to Kubernetes resource manifests, ex. Role, Subscription. These supplement or override defaults generated by run/cleanup
       --timeout duration          [olm only] Time to wait for the command to complete before failing (default 2m0s)
+      --olm-namespace string      [olm only] OLM Namespace (default "olm")
```

Comment:
Why it needs to be added to the cleanup command?

Reply:
when the run command is used with a non-default olm namespace  _`(olm run --olm-namesapce)`_, the registry server is setup in this namespace , so the cleanup commands needs this namspace when to delete the created resources from this namespace

Reply:
Why the cleanup will delete resources in the OLM-namespace? Should not the cleanup just delete the operator resources?

Reply:
currently, when the run command is executed
```
$ operator-sdk run --olm --operator-version 0.1.0 --manifests ~/pingpong-operator --olm-namespace openshift-operator-lifecycle-manager
```
it creates these resources in the olm namespace

```
...
INFO[0001] Creating registry
INFO[0001]   Creating ConfigMap "openshift-operator-lifecycle-manager/pingpong-operator-registry-bundles"
INFO[0001]   Creating Deployment "openshift-operator-lifecycle-manager/pingpong-operator-registry-server"
INFO[0001]   Creating Service "openshift-operator-lifecycle-manager/pingpong-operator-registry-server"
INFO[0001] Waiting for Deployment "openshift-operator-lifecycle-manager/pingpong-operator-registry-server" rollout to complete
INFO[0001] Waiting for Deployment "openshift-operator-lifecycle-manager/pingpong-operator-registry-server" to rollout: waiting for deployment spec update to be observed
...
---
 $ k get deployment
NAME                                READY   UP-TO-DATE   AVAILABLE   AGE
catalog-operator                    1/1     1            1           40d
olm-operator                        1/1     1            1           40d
packageserver                       2/2     2            2           40d
pingpong-operator-registry-server   1/1     1            1           41s
---
$ k get pods
NAME                                                 READY   STATUS    RESTARTS   AGE
catalog-operator-5d644f7b4b-p6vhj                    1/1     Running   0          40d
olm-operator-6d454db9dd-psnsz                        1/1     Running   0          40d
packageserver-647ff97ccd-z5nzs                       1/1     Running   0          7d19h
packageserver-647ff97ccd-z825v                       1/1     Running   0          7d19h
pingpong-operator-registry-server-7b8b4b8dc7-l9gxm   1/1     Running   0          51s
```
which is for the operator. So the cleanup has to delete these resources.